### PR TITLE
Moved additional Linux commands underneath...

### PR DESCRIPTION
**Pull request by BClark09**
...the appropriate heading.

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 1]
```diff
@@ -9,51 +9,47 @@ The binding uses pcap4j (https://www.pcap4j.org/) in order to capture ARP and BO
 
 ## Install Native Library
 
-Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J.
+Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J. Pcap4J needs administrator/root privileges.
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: Now you know why I always put one sentence per line ;)

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 2]
```diff
@@ -9,51 +9,47 @@ The binding uses pcap4j (https://www.pcap4j.org/) in order to capture ARP and BO
...
+Note: You can run Pcap4J with a non-root user by granting capabilities `CAP_NET_RAW` and `CAP_NET_ADMIN`
+to your java command by the following command: 
 
 ```
-yum install libpcap-devel
+sudo setcap cap_net_raw,cap_net_admin=eip `realpath /usr/bin/java`
 ```
 
-### Mac
+Be aware of other capabilities which are set by setcap. **These capabilities will be overwritten!** If you need more capabilities (like "cap_net_bind_service" for the Network binding), you have to add them like this :
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: Looks good. "Be aware of previously set capabilities"? Maybe mention getcap (see Network binding) to give the user the ability to actually check him/herself?

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 3]
```diff
@@ -9,51 +9,47 @@ The binding uses pcap4j (https://www.pcap4j.org/) in order to capture ARP and BO
 
 ## Install Native Library
 
-Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J.
+Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J. Pcap4J needs administrator/root privileges.
 
-### Ubuntu
+Source: [https://www.pcap4j.org/](https://www.pcap4j.org/)
+
+### Debian/Ubuntu
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: As we were talking about in the other thread. I'd reduce the "big" headlines count. How about:

H3 OS Specific Notes
H4 Debian/Ubuntu
H4 CentOS
...

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 4]
```diff
@@ -9,51 +9,47 @@ The binding uses pcap4j (https://www.pcap4j.org/) in order to capture ARP and BO
...
-### Mac
+Be aware of other capabilities which are set by setcap. **These capabilities will be overwritten!** If you need more capabilities (like "cap_net_bind_service" for the Network binding), you have to add them like this :
 
 ```
-brew install libpcap
+sudo setcap 'cap_net_raw,cap_net_admin=+eip cap_net_bind_service=+ep' `realpath /usr/bin/java`
 ```
 
-### Windows
+### CentOs
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: CentO**S**

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 5]
```diff
@@ -9,51 +9,47 @@ The binding uses pcap4j (https://www.pcap4j.org/) in order to capture ARP and BO
...
 
-Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J.
+Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J. Pcap4J needs administrator/root privileges.
 
-### Ubuntu
+Source: [https://www.pcap4j.org/](https://www.pcap4j.org/)
+
+### Debian/Ubuntu
 
 ```
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: Would be great if you could mark all code fences explicitly, e.g.:

    ```shell

Will ensure correct presentation, also part of remark-lint: https://github.com/wooorm/remark-lint/blob/master/doc/rules.md#fenced-code-flag

**Comment by BClark09** [comment]
@ThomDietrich Revised! 

@OLibutzki may want to confirm these changes also.

**Comment by OLibutzki** [comment]
Confirmed!

**Pull request merged**